### ISSUE TITLE
Managing bias at Facebook

### ISSUE THREAD

**Issue by dgroch** [rails-oceania/melbourne-ruby#38]
https://managingbias.fb.com

**Comment by gonzalo-bulnes** [comment]
On the same topic: [Unlocking the Clubhouse](https://mitpress.mit.edu/books/unlocking-clubhouse) reports on a 5-year research about minorities in computing which led to significant changes in the Carnegie Mellon curriculum, with lots of concrete examples, interviews, ideas, and numbers (for those who like them).

While it is primarily focused on _Women in Computing_, I found the report content deeply relevant far beyond gender-scoped considerations and felt identified with many of the experiences described by the interviewees.

**TL;DR** Worth reading for anyone interested in concrete steps that can be taken to favour diversity in computing.

**Comment by quamen** [comment]
I'm closing this as it's a link and not a talk per se. If people want to continue the discussion feel free to post these links to the meetup group's discussion board.
